set-env.ts

```diff
@@ -0,0 +1,40 @@
+import { writeFile } from 'fs';
+
+const environmentFilePath = `./src/environments/environment.ts`;
+const environmentFile = `
+export const environment = {
+  production: ${process.env['PRODUCTION'] || false},
```

Comment:
this info is duplicated in `const CONFIG = {...}` in `server.ts`. We would have to update in 2 places now when we add some new config. Is it possible to remove somehow this duplication?

Reply:
removed the duplication